hack/test-tools.sh

```diff
@@ -12,7 +12,12 @@ cd "${OS_ROOT}"
 source "${OS_ROOT}/hack/util.sh"
 source "${OS_ROOT}/hack/cmd_util.sh"
 source "${OS_ROOT}/hack/lib/test/junit.sh"
-os::log::install_errexit
+source "${OS_ROOT}/hack/lib/util/trap.sh"
+source "${OS_ROOT}/hack/lib/log/stacktrace.sh"
+
+trap os::test::junit::reconcile_output EXIT
```

Comment:
This looks like a fix that is not directly related to the other changes, or am I mistaken?